### PR TITLE
CC fixes

### PR DESCRIPTION
**Pull request by pavel-kokolemin**

### CODE REVIEW

File: BlocksettleNetworkingLib/Wallets/SyncWalletsManager.cpp [hunk 1]
```diff
@@ -1923,29 +1924,34 @@ bs::core::wallet::TXSignRequest WalletsManager::createPartialTXRequest(uint64_t
...
          for (const auto &recipient : recipients.second) {
-            txSize += recipient->getSize();
+            baseSize += recipient->getSize();
          }
       }
-      auto weight = 4 * txSize + witnessSize;
-      prevPartTxSize = (weight + 3) / 4;
-   }
+      // CC output
+      auto rec = std::make_shared<Recipient_P2WPKH>(CryptoPRNG::generateRandom(20), 1000);
```
Comment (Ation) [inline BlocksettleNetworkingLib/Wallets/SyncWalletsManager.cpp:1947]: It is always 31. Why do you need to call a PRNG for that?

File: WalletsLib/CheckRecipSigner.cpp [hunk 2]
```diff
@@ -417,3 +417,22 @@ NetworkType getNetworkType()
    default:       return NetworkType::RegTest;
    }
 }
+
+uint64_t bs::estimateVSize(const Signer &signer)
```
Comment (Ation) [inline WalletsLib/CheckRecipSigner.cpp:421]: If this method used only once in tests it should be there.

File: BlocksettleNetworkingLib/Wallets/SyncWalletsManager.cpp (hunk 1 above)
Comment (pavel-kokolemin) [reply BlocksettleNetworkingLib/Wallets/SyncWalletsManager.cpp]: sure, fixed

File: WalletsLib/CheckRecipSigner.cpp (hunk 2 above)
Comment (pavel-kokolemin) [reply WalletsLib/CheckRecipSigner.cpp:421]: it's used on server too

**Review by Ation — APPROVED** [review]

**Pull request merged**